tests/explorer.api.tests/ApiTests.cs

```diff
@@ -242,6 +242,16 @@ public async Task FailWithInvalidApiKey()
                 });
         }
 
+        [Fact]
+        public async Task FailWithInvalidExplorationId()
+        {
+            await TestApi(
+                HttpMethod.Get,
+                $"{resultEndpoint}/c29191e7-408e-4395-bb81-660c47181661",
```

Comment:
I think that it would be helpful for the semantics to use GUID like "11111111-1111-1111-1111-111111111111".

Reply:
done 
4da4db3